### DISCUSSION TITLE
Hardware recommendations for 8-10 cameras. Minisforum, BMAX or Beelink?

### DISCUSSION THREAD

**Discussion by spantobi** [blakeblackshear/frigate#3426] [General]
Hi,

I'm running Frigate in vmware at the moment with a USB card passthrough for my Coral USB. The problem is that it crashes pretty much once per day. I'm guessing it's the passthrough doing something funky.

So I was thinking that I want to try running Frigate with a separate barebone machine. I see Bmax, Minisforum and Beelink being recommended in the official recommended hardware list. Which one do you think would be best for my setup?

Thanks for any advice!

**Comment by blakeblackshear** [comment]
It really depends on the resolution and frame rates that you will use for detection and how much activity you have on your cameras.

**Comment by spantobi** [comment]
Thanks for your reply. I am detecting at 1920x1080 and 5 fps. Activity is pretty high on half of the cameras usually.

**Reply by blakeblackshear** [reply]
I think you may want to look at something with a more powerful processor like an Intel NUC. It doesn't have to be the latest generation. The J4125 may be a little under powered for that many cameras.

**Reply by spantobi** [reply]
Ok! Good to know, I will look into a NUC I guess :)

**Reply by baylanger** [reply]
@spantobi I'm curious to know if the issue was with Frigate itself crashing or the vm?

You mind sharing if you ended up switching to barebone and if so, what you ended up getting, you camera config (4k or less & fps) as well as the load?

I've only setup 1 4k camera yet on my NAS but thinking to move Frigate/deepstack/doubletake to its own box.

**Reply by baylanger** [reply]
@spantobi I had not noticed your post above, you have 2MP cameras all @ 5fps. Just need to know how the load is doing and are  you only running Frigate there?
